src/layouts/components/CardContainer.js

```diff
@@ -2,31 +2,82 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
+import { media } from '../../theme/globalStyle'
+
 const StyledCardContainer = styled.div`
   display: grid;
   grid-template-columns: repeat(12, 1fr);
   grid-template-rows: auto;
   margin: 1.5rem 0 3rem 0;
+  ${media.desktop`
+    margin: 0;
+  `} ${media.phone`
+    margin: 0.5rem 0 1rem 0;
+  `};
 `
 
 const StyledStories = styled.div`
-  grid-column: 3 / span 8;
+  grid-column: 2 / span 10;
```

Comment:
Not seen this syntax before, can you tell me what it is doing please @mar-bi 🙏

Reply:
It's standard syntax of **CSS** Grid :upside_down_face: 
There are 2 ways to assign area for an item:
- using `grid-area` (you saw it before :upside_down_face:  )
- using `grid-column`, `grid-row`.
The last option takes 2 values: *start line / end line* or *start line / span (value)*
I prefer to use it with the `span`. 

Here `grid-column: 2 / span 10;` I give for a component area which starts at the **2-nd line** of the grid and extends for **10 columns**.
The same we can do for rows.